activity/scraper.py

```diff
@@ -1,7 +1,6 @@
 import requests
 import json
 import datetime
-import os
```

Comment:
unrelated to your issue ; please remove.  I've created https://github.com/coala/community/issues/62 for this.